lib/Backend/GlobOptFields.cpp

```diff
@@ -2934,6 +2933,7 @@ GlobOpt::SetObjectTypeFromTypeSym(StackSym *typeSym, const Js::Type *type, Js::E
     {
         JsTypeValueInfo* valueInfo = JsTypeValueInfo::New(this->alloc, type, typeSet);
         valueInfo->SetSymStore(typeSym);
+        this->SetSymStoreDirect(valueInfo, typeSym);
```

Comment:
Good catch, the old one should be removed.

---
In reply to: [67777712](https://github.com/Microsoft/ChakraCore/pull/1152#discussion_r67777712) [](ancestors = 67777712)